### DISCUSSION TITLE
Can't make 120hz work with @react-navigation/stack on iPhone

### DISCUSSION THREAD

**Discussion by vedmant** [react-navigation/react-navigation#11870] [Q&A]
I have an Expo app, it generally scrolls and works with 120hz. However only place where I clearly see it's 60hz is stack navigation animation.

I checked and can confirm that I have `<key>CADisableMinimumFrameDurationOnPhone</key><true/>`.

My stack is simple:

```
<Stack.Navigator screenOptions={{ headerShown: true }}>
    <Stack.Screen name="ListEntries" component={EntriesScreen} />
    <Stack.Screen name="EditEntry" component={EditEntryScreen} />
    <Stack.Screen name="AddEntry" component={AddEntryScreen} />
</Stack.Navigator>
```

Versions:
```
"expo": "~50.0.8",
"react-native": "0.73.4",
"@react-navigation/native": "^6.1.14",
"@react-navigation/stack": "^6.3.25",
```

**Comment by rohitx15** [comment]
Were you ever able to figure this out?

**Reply by vedmant** [reply]
No, still 60 hz for stack while other parts of app run 120hz.

**Reply by andiskim** [reply]
@rohitx15 @vedmant Have you guys been able to figure this out?

**Comment by SorooshDeveloper** [comment]
we also have this issue on ios.

**Comment by andiskim** [comment]
@rohitx15 @vedmant @SorooshDeveloper 

Hey guys, I was able to fix using [@react-navigation/native-stack](https://github.com/react-navigation/react-navigation/blob/main/packages/native-stack) instead of [@react-navigation/stack](https://github.com/react-navigation/react-navigation/blob/main/packages/stack)

**Comment by rohitx15** [comment]
Yep, same. It works with native-stack.
